### PR TITLE
setup and school lookup scripts

### PR DESCRIPTION
**Pull request by alice-hannah**

### CODE REVIEW

File: code/00_setup.R [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+## The label used to name the folder of output data from this run. 
+## This should be changed for every update to name the new output folder. 
+## Follow the naming convention: "year-month of publish" e.g. 2021-May 
+
+run_label <- "2022-July"
+
+
+### 2 - Dates - TO UPDATE ----
+
```
Comment (LauraStott) [inline code/00_setup.R:25]: Need to add a comment here about dogs

File: code/00_setup.R [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+## (e.g. acel) each file contains a full trend and supercedes previous files.
+
+
+## 2a - Datasets typically updated in April ----
+
+# Datasets that require a single file, update with a single year
+year_contacts <- 2022
+year_bge      <- 2021
+
+# Summary data requires multiple files but is only updated in odd-numbered
```
Comment (LauraStott) [inline code/00_setup.R:38]: comment change to: is typically only updated in odd-numbered years

File: code/00_setup.R [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+# years; e.g. 2021, 2023, etc.
+# Update the vector with the new year; e.g. c(2021, 2023)
```
Comment (LauraStott) [inline code/00_setup.R:40]: Update the vector (list) with the new year: e.g. c(2021, 2023)

File: functions/import_summary_data.R [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+#' 
+#' @export
+#'
+#' @examples
+#' import_summary_data("LA and SCOT", 2021)
+#' import_summary_data("LA and SCOT", "timeseries")
+
+import_summary_data <- function(sheet_name, calendar_year) {
+  
+  # Define pipe
```
Comment (LauraStott) [inline functions/import_summary_data.R:20]: I would need clearer explanation of what this does and I don't know :)

File: functions/import_summary_data.R [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  `%>%` <- magrittr::`%>%`
+  
+  filepath <- here::here("data", 
+                          "school_summary_statistics", 
+                          paste0(calendar_year, 
+                                 "_school_summary_statistics.xlsx"))
+  
+  # Check datafile exists and if not produce error
```
Comment (LauraStott) [inline functions/import_summary_data.R:28]: again I would need more clarification on these comments please

**Review by Laura-Dale — CHANGES_REQUESTED** [review]
Mostly just additions to comments from me

File: functions/import_summary_data.R (hunk 4 above)
Comment (alicebyers5) [reply functions/import_summary_data.R]: Is this better?

Comment (alicebyers5) [reply functions/import_summary_data.R]: Not sure if you can see it in the wee code preview but you should be able to see it if you go to the same line in Files Changed tab

File: functions/import_summary_data.R (hunk 5 above)
Comment (alicebyers5) [reply functions/import_summary_data.R]: I've expanded this too but let me know if still not clear

**Review by Laura-Dale — APPROVED** [review]

**Pull request merged**